### ISSUE TITLE
if you go to /year/month/day it says "loading" perpetually 

### ISSUE THREAD

**Issue by ufxela** [TheStanfordDaily/stanforddaily#119]
e.g. https://www.stanforddaily.com/2020/05/22/

would be cool to show all articles published on that day?

**Comment by epicfaace** [comment]
Yes -- either that or we could also show that day's digest (like the day's headlines basically)

@hs4man21
